pages/c/_cluster/_product/projectsnamespaces.vue

```diff
@@ -29,7 +29,8 @@ export default {
       return;
     }
 
-    this.rows = await this.$store.dispatch(`${ inStore }/findAll`, { type: NAMESPACE });
+    this.rows = (await this.$store.dispatch(`${ inStore }/findAll`, { type: NAMESPACE }))

```

Comment:
The reactivity of the list is lost if we de-reference from the store (new ns created externally don't show up, inline deleted ns aren't removed). Fetching all namespaces up front and then using a computed method with a getter for namespaces (+ filter) seems to work. 

```
    await this.$store.dispatch(`${ inStore }/findAll`, { type: NAMESPACE });
....
    rows() {
      return this.$store.getters[`${ inStore }/all`](NAMESPACE).filter(namespace => !namespace.isObscure);
    },
```

(had to play around with this, thought it would be easier)

Reply:
Good catch.